compat/test/browser/render.test.js

```diff
@@ -158,6 +158,43 @@ describe('compat render', () => {
 		expect(scratch.firstElementChild.value).to.equal('0');
 	});
 
+	it('should call onChange and onInput when input event is dispatched', () => {
+		const onChange = sinon.spy();
+		const onInput = sinon.spy();
+
+		render(<input onChange={onChange} onInput={onInput} />, scratch);
+
+		scratch.firstChild.dispatchEvent(createEvent('input'));
+
+		expect(onChange).to.be.calledOnce;
+		expect(onInput).to.be.calledOnce;
+
+		onChange.resetHistory();
+		onInput.resetHistory();
+
+		// change props order
+		render(<input onInput={onInput} onChange={onChange} />, scratch);
+
+		scratch.firstChild.dispatchEvent(createEvent('input'));
+
+		expect(onChange).to.be.calledOnce;
+		expect(onInput).to.be.calledOnce;
+	});
+
+	it('should call onChange and onInput even when one of the callbacks failed', () => {
+		const onChange = sinon.spy(() => {
+			throw new Error();
```

Comment:
Dug into the failing coverage and the test suite errors out here because it throws in the event handler, maybe JSDom can't handle that well

Reply:
@JoviDeCroock Pushed a fix to allow uncaught error in mocha

Reply:
A lot of the tests seem to be failing due to not finding the node internals https://github.com/preactjs/preact/runs/6866093495?check_suite_focus=true#step:6:160